src/julia-parser.scm

```diff
@@ -234,7 +234,7 @@
 
 (define (read-operator port c)
   (if (and (eqv? c #\*) (eqv? (peek-char port) #\*))
-      (error "use \"^\" instead of \"**\""))
```

Comment:
Maybe just `use "x^y" instead of "x**y" for exponentiation`

Reply:
Do you think this is the right approach for this improvement, rather than making `**` it's own operator?

Reply:
We're not going to make `**` the exponentiation operator in Julia…

Reply:
fair enough.

Reply:
See above: maybe the shorter: `use "x^y" instead of "x**y" for exponentiation, and "x..." instead of "**x" for splatting`